### DISCUSSION TITLE
How to "emitTo" event from livewire component to Filament page

### DISCUSSION THREAD

**Discussion by Justpaolo** [livewire/livewire#5619] [1. Help] [answered]
I have a livewire component inside a Filament page. 

I'd like to emit an event using emitTo (always my first choice) but I don't know how to address the filament page.

This is not a blocking problem, emitUp will do the work, anyway I'd like to know if it's possible (and, in case, how) to use emitTo with component (filament pages in my case) outside the livewire directory.

Thanks in advance

**Comment by joshhanley** [comment, marked as answer]
@Justpaolo I've not really used Filament so am not sure how to do it. I'd suggest asking the question on their Discord as they're more likely to know the answer https://filamentphp.com/discord

Hope this helps!
